action/protocol/staking/protocol.go

```diff
@@ -219,6 +224,18 @@ func (p *Protocol) CreatePreStates(ctx context.Context, sm protocol.StateManager
 		}
 	}
 
+	if !p.voteReviser.NeedRevise(blkCtx.BlockHeight) {
+		return nil
+	}
```

Comment:
this is incorrect. should just skip revise not the whole function. that is why i was to put this as an internal function.